starboard/build/config/BUILDCONFIG.gn

```diff
@@ -408,12 +408,21 @@ template("evergreen_loader") {
   }
   copy("copy_loader_app_lib${original_target_name}") {
     install_content = true
+
+    # TODO(b/348401006): remove this special case when we support compression
+    # for all targets.
+    if (original_target_name == "cobalt") {
+      library_extension = "lz4"
+    } else {
+      library_extension = "so"
+    }
+
     if (target_cpu == "arm" && arm_float_abi == "softfp") {
-      sources = [ "$root_out_dir/../evergreen-$target_cpu-${arm_float_abi}_$build_type/install/lib/lib${original_target_name}.so" ]
+      sources = [ "$root_out_dir/../evergreen-$target_cpu-${arm_float_abi}_$build_type/install/lib/lib${original_target_name}.${library_extension}" ]
     } else if (target_cpu == "arm64") {
-      sources = [ "$root_out_dir/../evergreen-$target_cpu_$build_type/install/lib/libcobalt.so" ]
+      sources = [ "$root_out_dir/../evergreen-$target_cpu_$build_type/install/lib/lib${original_target_name}.${library_extension}" ]
```

Comment:
Nit: I think this is slightly clearer. You choose how you want it.

```
    abi_suffix = ""
    if (target_cpu == "arm" && arm_float_abi == "softfp") {
      abi_suffix = "-${arm_float_abi}"
    }
    sources = [ "$root_out_dir/../evergreen-$target_cpu${abi_suffix}_$build_type/install/lib/lib${original_target_name}.${library_extension}" ]
```

Reply:
Agreed, thanks.